### PR TITLE
chore: improve README.md

### PR DESCRIPTION
**Pull request by EnokiUN**

### CODE REVIEW

**Review by GreenArron — APPROVED** [review]
smh not enough 🚀 😔 \j (👍)

File: README.md [hunk 1]
```diff
@@ -1,9 +1,21 @@
...
+
+## Deployment
+
+We really recommend and *only* officially support using the provided docker-compose as a quick way to get stuff running, just edit your `Eludris.toml` to suit your needs then run
+
+```sh
+docker-compose up
+```
+
+Congratulations, you've now successfully deployed your Eludris instance!
```
Comment (teaishealthy) [inline README.md:13]:
```suggestion
Congratulations, you've now successfully deployed your Eludris instance! 🚀
```

**Review by teaishealthy — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (EnokiUN) [reply README.md:13]: Now, this minor change had rather huge consequences on the foreseeable future of Eludris as it drastically changes the way Eludris is marketed and the target audience more likely to use it, I'm not quite sure if that's desirable yet which is why I might open a DEP soon, for now I'll let the other members vote on this change.

TLDR: rocket emoji cringe

Comment (GreenArron) [reply README.md:13]: but is it the Rust way to not use the sacred "🚀"

File: README.md [hunk 2]
```diff
@@ -1,9 +1,21 @@
...
 
 ## Default Ports
 
 Oprish: 7159
```
Comment (ooliver1) [inline README.md:17]: As a new user has no idea what these are when looking at the README alone, perhaps add links to these names so they know one is http, another is ws and another is a cdn

File: README.md (hunk 1 above)
Comment (EnokiUN) [reply README.md:13]: > "Eludris needs trump any Rust needs"
\- Not Linus Torvalds

**Review by ooliver1 — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (EnokiUN) [reply README.md:17]: Great idea, perhaps now is the time to start MDBook based Eludris docs which aim to explain everything Eludris related + API docs?

**Review by GreenArron — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (teaishealthy) [reply README.md:13]: You could not be more wrong. The rocket emoji here does not symoblize Eludris speed or a similarly outrageous statement, it is meant to make the user really feel their accomplisment of finally installing Eludris and switching to a better life.

\- tea :rocket:

Comment (EnokiUN) [reply README.md:13]: How about instead of cringe rocket emoji we make a tiny thang emoji with confetti?

Comment (EnokiUN) [reply README.md:13]: might actually do that tomorrow when I have time then upload it to tooty effis static when I add that

**Review by ooliver1 — APPROVED** [review]
owo

**Review by alvaro-sch — APPROVED** [review]

**Review by GreenArron — APPROVED** [review]

**Review by teaishealthy — CHANGES_REQUESTED** [review]
rocket
